docs/using.rst

```diff
@@ -163,6 +163,21 @@ manager.
 Both relative and absolute paths work for Python 3.7 and newer.
 
 
+Migrating from Legacy
+=====================
+
+Starting with Python 3.9 and ``importlib_resources`` 1.4, this package
+introduced the ``files()`` API, to be preferred over the legacy API,
+i.e. the functions ``open_binary``, ``open_text``, ``path``,
+``contents``, ``read_text``, ``read_binary``, and ``is_resource``.
+
+To port to the ``files()`` API, refer to the
+`_legacy module <https://github.com/python/importlib_resources/blob/66ea2dc7eb12b1be2322b7ad002cefb12d364dff/importlib_resources/_legacy.py>`_
+to see simple wrappers that enable drop-in replacement based on the
+preferred API, and either copy those or adapt the usage to utilize the
+``files`` and ``Traversable`` interfaces directly.
```

Comment:
I think we should link to the protocol description here. Unfortunately, the stdlib documentation does not show the methods, but we can fix that.

https://docs.python.org/3/library/importlib.html#importlib.abc.Traversable

Reply:
Addressed in 239ae6d56a.